views/subscribe.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="container mt-5">
+  <div class="col-8 offset-2 text-center">
+
+
+    <h2>Auf dem Laufenden bleiben</h2>
+    <p>Trag dich in unseren Newsletter ein! Wir verschicken ihn 1-2 Mal im Monat. Und hier kannst du alle Newsletter
+      bis
+      Mai 2018 nachlesen: <a target="_blank" href="http://us10.campaign-archive1.com/home/?u=63598878eb7adee616350bca7&amp;id=9457fbbf4c">Lernwerkstatt-Archiv</a></p>
+  </div>
+</div>
+{{> subscribe/mailchimp this }}
```

Comment:
I think, mailchimp partial doesn't do anything with an empty **this** object. :)